test/pedestal_toolbox/params_test.clj

```diff
@@ -102,3 +102,14 @@
                               {:date "4 score and 20 years ago"}}}
                             enter
                             (get-in [:response :body])))))))))
+
+(deftest query-param-accept-test
+  (let [params {"a" 1 :accept "application/csv"}
+        ctx {:request {:params params}}
+        enter (:enter query-param-accept)
+        ctx-with-updated-headers (enter ctx)]
+    (testing "a query param of content type is placed in the headers"
+      (is (= (get-in ctx [:request :params :accept])
```

Comment:
I'd rather see a literal comparison value, or a reference to a let'd string of just `"application/csv"` as this test's expected value. Otherwise the more indirection you have in your expected value, the more likely you are to have passing tests where both the expected and actual end up being blank, nil, or some other value you don't want. That seems very unlikely here, but I like to draw bright lines around things. :)